src/crimson/os/seastore/btree/btree_range_pin.h

```diff
@@ -336,14 +358,16 @@ class btree_pin_set_t {
   }
 
 public:
+  btree_pin_set_t() : pins(8) {}
```

Comment:
Magic number, and is the max depth 8 is appropriate?

Reply:
I calculated this, for a 8-depth btree in which each node has a minimum number of entries(70 entries each node), it can hold 576480100000000 entries. If each entry represents a 4KB block, it can index a 2PB disk space. I thought this should be enough.

Reply:
I think I'm only using 7 of the 8 vector entries, will correct that

Reply:
We'll need to aware this if need to expand the tree value in the future.

Let's define a MAX_FIXEDKVBTREE_DEPTH instead, and use `ceph_assert` to make sure the tree won't exceed this depth limit in the root split operation.

Is it reasonable to you?

Reply:
Yeah, will do.